packages/shared/routes/dashboard/settings/views/advanced/NetworkConfiguration.svelte

```diff
@@ -1,36 +1,27 @@
 <script lang="typescript">
-    import { Button, Checkbox, HR, Radio, Text, NodeConfigOptions } from 'shared/components'
     import { localize } from '@core/i18n'
     import {
         getNodeCandidates,
-        isOfficialNetwork,
-        updateClientOptions,
+        getOfficialNodes,
         INode,
-        NETWORK_HEALTH_COLORS,
-        NetworkStatusDescription,
-        networkStatus,
+        isOfficialNetwork,
         NetworkHealth,
-        getOfficialNodes,
-        nodeInfo,
+        networkStatus,
+        NetworkStatusDescription,
         NetworkType,
+        NETWORK_HEALTH_COLORS,
+        nodeInfo,
     } from '@core/network'
+    import { activeProfile } from '@core/profile'
     import { closePopup, openPopup } from '@lib/popup'
-    import { activeProfile, updateActiveProfile } from '@core/profile'
+    import { Button, Checkbox, HR, NodeConfigOptions, Radio, Text } from 'shared/components'
 
     let contextPosition = { x: 0, y: 0 }
     let nodeContextMenu: INode
     let nodesContainer
 
     $: clientOptions = $activeProfile?.clientOptions
 
-    $: {
```

Comment:
Removing all this piece of code from here will make the rest of the code below that edits `clientOptions` not work, right? For example, 
```ts
    function handleManualNodeSelection() {
        clientOptions.includeOfficialNodes = true
    }
```
will not trigger any update

Reply:
Not sure because this stuff isn't enabled in the developer mode. The node config needs refactoring as is!

Reply:
Added a task for this: https://github.com/iotaledger/firefly/issues/4165